README.md

```diff
@@ -16,7 +18,7 @@ You are going to create a Dockerfile for a simple web application, built it, get
 
 ## Create a local repo for your app
 
-For this tutorial you will use a very simple Flask application, written in python, and create a production ready docker image.
+For this tutorial you will use a very simple Flask application, written in python, and create a production ready Docker image.
```

Comment:
Thanks I will fix the Docker in the future (I totally do not see the missing capital after writing the word this many times !!)